inst/include/interface/rcpp/rcpp_objects/rcpp_fleet.hpp

```diff
@@ -667,10 +552,6 @@ class FleetInterface : public FleetInterfaceBase {
       FIMS_INFO_LOG(" adding Fleet length object to TMB out loop");
```

Comment:
Can some of these FIMS_INFO_LOG messages be removed? They read like they were put in by a developer trying to debug (lines 625, 627, 648, 665, 667).